runtime/query.go

```diff
@@ -6,24 +6,25 @@ import (
 	"strings"
 
 	"github.com/rilldata/rill/runtime/pkg/observability"
-	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
 )
 
-var (
-	meter                   = global.Meter("runtime")
-	queryCacheHitsCounter   = observability.Must(meter.Int64Counter("query_cache.hits"))
-	queryCacheMissesCounter = observability.Must(meter.Int64Counter("query_cache.misses"))
-)
+type QueryResult struct {
+	Value any
+	Bytes int64
+}
+
+var queryCacheEntrySizeHistogram = observability.Must(meter.Int64Histogram("query_cache.entry_size", metric.WithUnit("bytes")))
```

Comment:
Move to the other metrics to keep things together. It can just pass the `queryName` as a param to `getOrLoad` for the attributes.